dev-libs/zziplib/zziplib-0.13.69.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit libtool eutils flag-o-matic python-any-r1
+
+DESCRIPTION="Lightweight library for extracting data from files archived in a single zip file"
+HOMEPAGE="https://github.com/gdraheim/zziplib"
+SRC_URI="https://github.com/gdraheim/zziplib/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="|| ( LGPL-2.1 MPL-1.1 )"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86"
+IUSE="doc sdl static-libs test"
+
+RDEPEND="
+	sys-libs/zlib
+	sdl? ( >=media-libs/libsdl-1.2.6 )"
+DEPEND="${RDEPEND}
+	${PYTHON_DEPS}
+	virtual/pkgconfig
+	test? ( app-arch/zip )"
+
+src_prepare() {
+	python_fix_shebang .
+
+	# workaround AX_CREATE_PKGCONFIG_INFO bug #353195
+	sed -i \
+		-e '/ax_create_pkgconfig_ldflags/s:$LDFLAGS::' \
+		-e '/ax_create_pkgconfig_cppflags/s:$CPPFLAGS::' \
+		configure || die
+
+	# zziplib tries to install backwards compat symlinks we dont want
+	sed -i -e '/^zzip-postinstall:/s|$|\ndisable-this:|' Makefile.in || die
+	sed -i -e '/^install-exec-hook:/s|$|\ndisable-this:|' zzip/Makefile.in || die
+
+	elibtoolize
+
+	# Do an out-of-tree build as their configure will do it automatically
+	# otherwise and that can lead to funky errors. #492816
+	mkdir -p build
```

Comment:
Yes, it fails without that:
```
>>> Configuring source in /tmp/portage/dev-libs/zziplib-0.13.69/work/zziplib-0.13.69 ...
 * econf: updating zziplib-0.13.69/uses/config.sub with /usr/share/gnuconfig/config.sub
 * econf: updating zziplib-0.13.69/uses/config.guess with /usr/share/gnuconfig/config.guess
/tmp/portage/dev-libs/zziplib-0.13.69/work/zziplib-0.13.69/configure --prefix=/usr --build=x86_64-pc-linux-gnu --host=x86_64-pc-linux-gnu --mandir=/usr/share/man --infodir=/usr/share/info --datadir=/usr/share --sysconfdir=/etc --localstatedir=/var/lib --disable-dependency-tracking --disable-silent-rules --docdir=/usr/share/doc/zziplib-0.13.69 --htmldir=/usr/share/doc/zziplib-0.13.69/html --with-sysroot=/ --libdir=/usr/lib64 --disable-sdl --disable-static
configure: loading site script /usr/share/config.site
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-pc-linux-gnu
checking target system type... x86_64-pc-linux-gnu
continue configure in default builddir "./x86_64-pc-linux-gnu"
....exec /bin/sh ..//tmp/portage/dev-libs/zziplib-0.13.69/work/zziplib-0.13.69/configure "--srcdir=.." "--enable-builddir=x86_64-pc-linux-gnu" "linux
gnu"
/bin/sh: ..//tmp/portage/dev-libs/zziplib-0.13.69/work/zziplib-0.13.69/configure: No such file or directory
```
It seems it might be due the `srcdir=..`, since with that line and the one over line 50 you end in a directory above the configure file.